Frontend-v1-Original/components/ssBribeCreate/ssBribeCreate.tsx

```diff
@@ -381,6 +301,183 @@ export default function ssBribeCreate() {
   );
 }
 
+function GaugeSelect({
+  value,
+  gaugeOptions,
+  onSelect,
+}: {
+  value: Gauge | null;
+  gaugeOptions: Gauge[];
+  onSelect: (value: Gauge) => void;
+}) {
+  const [open, setOpen] = useState(false);
+  const [search, setSearch] = useState("");
+
+  const openSearch = () => {
+    setOpen(true);
+    setSearch("");
+  };
+
+  const onClose = () => {
+    setSearch("");
+    setOpen(false);
+  };
+
+  const onSearchChanged = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearch(event.target.value);
+  };
+
+  const onLocalSelect = (gauge: Gauge) => {
+    setSearch("");
+    setOpen(false);
+    onSelect(gauge);
+  };
+
+  const filteredGaugeOptions = useMemo(() => {
+    const go = gaugeOptions.filter((gauge) => {
+      if (search && search !== "") {
+        return (
+          gauge.address.toLowerCase().includes(search.toLowerCase()) ||
+          gauge.token0.symbol.toLowerCase().includes(search.toLowerCase()) ||
+          gauge.token1.symbol.toLowerCase().includes(search.toLowerCase())
+        );
```

Comment:
```suggestion
        return [gauge.address, gauge.token0.symbol, gauge.token1.symbol].some(
          (s) => s.toLowerCase().includes(search.trim())
        );
```
just nitpicking here.